### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by nglgzz**
### Please check if the PR fulfills these requirements
- [x] The commit message follows our [guidelines](https://github.com/neo4j-devtools/relate/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?
Update dependencies. The biggest changes are:

- Upgrade to `@oclif/core`. Most of the oclif packages have been deprecated and merged into `@oclif/core`, along with some other breaking changes. I followed this [migration guide](https://github.com/oclif/core/blob/main/MIGRATION.md) for the upgrade.
- Upgrade to Lerna v5.
- Upgrade GraphQL ([migration guide](https://docs.nestjs.com/graphql/migration-guide)).
- Upgrade NestJS and related packages ([migration guide](https://docs.nestjs.com/graphql/migration-guide)).
- Fix NPM audit issues.

### What is the current behavior?
-

### What is the new behavior?
There is only one change in behavior which is, space-separated commands are now supported. This change is backward compatible and colon-separated commands work as they did before too. So for example:

```
relate dbms:start movies
```

can now be also run as: 

```
relate dbms start movies
```

### Does this PR introduce a breaking change?
Yes, the ranges for the peer dependencies on NestJS have been updated.

### CODE REVIEW

File: packages/cli/src/help/index.ts [hunk 1]
```diff
@@ -21,51 +20,12 @@ ${oclifHelp}`;
...
+    formatTopic(topic: Interfaces.Topic): string {
         const oclifHelp = super.formatTopic(topic);
 
         return `${fromViaToPattern('relate', 'topic', topic.name)}
 
 ${oclifHelp}
     `;
     }
-
-    public showCommandHelp(command: Command) {
```
Comment (linuslundahl) [inline packages/cli/src/help/index.ts:33]: This is not needed anymore?

**Review by linuslundahl — APPROVED** [review]
Good stuff! 👍

File: packages/cli/src/help/index.ts (hunk 1 above)
Comment (nglgzz) [reply packages/cli/src/help/index.ts:33]: It's built into oclif now

Comment (linuslundahl) [reply packages/cli/src/help/index.ts:33]: Yeah, that's what I thought. 👍

**Review by linuslundahl — APPROVED** [review]

**Pull request merged**